e2e/provision/gce_init.sh

```diff
@@ -56,6 +56,7 @@ runuser -u $NEPHIO_USER ./gce_install_sandbox.sh
 # Grant Docker permissions to current user
 if ! getent group docker | grep -q "$NEPHIO_USER"; then
     sudo usermod -aG docker "$NEPHIO_USER"
+    newgrp docker
```

Comment:
I don't think this will work, or at least not as expected. Basically, [it will create a subshell during the execution](https://unix.stackexchange.com/a/18902).

Usually the way to use it in a shell is like the follows:

```bash
$ newgrp docker <<EONG
docker ps
EONG
```
instead of:
```bash
$ sudo docker ps
```